specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/ApplicationGroups.json

```diff
@@ -0,0 +1,329 @@
+        "x-ms-examples": {
+          "ApplicationGroupGet": {
+            "$ref": "./examples/ApplicationGroup/ApplicationGroupGet.json"
+          }
+        },
+        "description": "Gets an ApplicationGroup for a Namespace.",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/NamespaceNameParameter"
+          },
+          {
+            "$ref": "../../../common/v2/definitions.json#/parameters/ApplicationGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Namespace ApplicationGroup returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/AuthorizationRule"
+            }
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ApplicationGroupListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationGroup"
+          },
+          "description": "Result of the List Application Groups operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains an incomplete list of Authorization Rules"
+        }
+      },
+      "description": "The response from the List Application Groups operation."
+    },
+    "ApplicationGroup": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "updatedAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was updated"
+            },
+            "createdAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was created."
+            },
+            "isEnabled": {
+              "type": "boolean",
+              "description": "Active connection status from client to Event hub"
+            },
+            "appGroupIdentifier": {
+              "readOnly": true,
+              "type": "string",
+              "description": "The Application Group Identifier"
```

Comment:
This needs more description. It should clearly state what is the expected format of an appIdentifier and which auth Keys are supported with authKeyname.